src/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './formatters';
 export { SimpleTextEditor } from './editors';
 export * from './enums';
 export * from './types';
+export { useFocusRef } from './hooks';
```

Comment:
Don't know if it worth to export.

Reply:
You are probably right. I have removed it for now